File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L3/01/L3.py
# Import necessary libraries
import math

# Define the main function
def main():
    mil = 0
    print("Miles\t\tKilometers")
    print("-------------------------------")
    
    mil = 1
    while mil <= 10:
        print(f"{mil}\t\t{mil * 1.609:.3f}")
        mil += 1

# Call the main function
if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L3/01/L3.kt
package IR

fun main() {
    var mil = 0
    println("Miles\t\tKilometers")
    println("-------------------------------")
    mil = 1
    while (mil <= 10) {
        println("$mil\t\t${mil * 1.609}")
        mil++
    }
}

